README.md

```diff
@@ -11,6 +11,17 @@ API Types for Rancher 2.0
 
 Run `go generate` in the root of the project
 
+### Running code generation after go modules
+
+The code generation code still depends on the GOPATH. 
+To generate types you need to update vendor.
+
+1. Update go.mod for what you need 
+2. Run `GO111MODULE=on go mod vendor`
+3. Export your gopath (user must export the GOPATH env variable)
```

Comment:
In the parenthesis here you could just write the command instead: `export GOPATH=dir_to_types_go_path`